src/Components/LifecycleTable/LifecycleTable.tsx

```diff
@@ -218,9 +216,10 @@ export const LifecycleTable: React.FunctionComponent<LifecycleTableProps> = ({ d
 
   const renderSystemLifecycleData = () => {
     return (paginatedRows as SystemLifecycleChanges[]).map((repo: SystemLifecycleChanges) => {
-      if (!repo.name || !repo.count) {
+      if (!repo.name || !repo.release_date || !repo.retirement_date || !repo.count) {
```

Comment:
This hides any data from the other source that might try to sneak in during state updates.